packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -46,10 +46,8 @@ import { toArray } from '@lumino/algorithm';
 import { IDragEvent } from '@lumino/dragdrop';
 
 import '@elyra/canvas/dist/common-canvas.min.css';
-
-import '@elyra/canvas/dist/common-canvas.min.css';
-
 import 'carbon-components/css/carbon-components.min.css';
+import '../style/canvas.css';
```

Comment:
I assume this change is because you're editing the file to support dark mode? Is there a way to override the css value in our own file rather than copying and editing a external library's one? If not could you annotate the diff in this PR detailing what you changed in the file?

Reply:
this is not an external file from canvas. i created the new `canvas.css` file just to keep all the changes needed for canvas in a single document. i could add them into the existing `index.css` instead but i believe it is better left separate. and if canvas does change it default theme or adds support for dynamically switching themes, most of the work needed for Elyra would be contained in this file.

Reply:
Why are you removing `import '@elyra/canvas/dist/common-canvas.min.css';` above then. I had assumed that this replaced that since it was deleted. I agree though that keeping canvas.css separate from index.css is a good idea in this case.

Reply:
the line:
```
import '@elyra/canvas/dist/common-canvas.min.css'
```
was actually included twice ([line 48 and 50](https://github.com/elyra-ai/elyra/blob/master/packages/pipeline-editor/src/PipelineEditorWidget.tsx#L48-L50)) so i removed the duplicate.

Reply:
oh >.< nice catch